packages/boxel-ui/addon/styles/variables.css

```diff
@@ -15,21 +15,21 @@
     var(--boxel-font-family);
 
   /* font-sizes */
-  --boxel-font-size-xl: 2rem;
-  --boxel-font-size-lg: 1.375rem;
-  --boxel-font-size-med: 1.25rem;
+  --boxel-font-size-xl: 2rem; /* 32px */
+  --boxel-font-size-lg: 1.375rem; /* 22px */
+  --boxel-font-size-med: 1.25rem; /* 20px */
   --boxel-font-size: 1rem; /* default - 16px */
-  --boxel-font-size-sm: 0.8125rem;
-  --boxel-font-size-xs: 0.6875rem;
+  --boxel-font-size-sm: 0.8125rem; /* 13px */
+  --boxel-font-size-xs: 0.6875rem; /* 11px */
 
   /* letter-spacing */
-  --boxel-lsp-xxl: 0.1em;
-  --boxel-lsp-xl: 0.05em;
-  --boxel-lsp-lg: 0.035em;
-  --boxel-lsp: 0.025em;
-  --boxel-lsp-sm: 0.015em;
-  --boxel-lsp-xs: 0.01em;
-  --boxel-lsp-xxs: 0.005em;
+  --boxel-lsp-xxl: 0.1em; /* 1.6px */
+  --boxel-lsp-xl: 0.05em; /* 0.8px */
+  --boxel-lsp-lg: 0.035em; /* 0.56px */
+  --boxel-lsp: 0.025em; /* 0.4px */
+  --boxel-lsp-sm: 0.015em; /* 0.24px */
+  --boxel-lsp-xs: 0.01em; /* 0.16px */
+  --boxel-lsp-xxs: 0.005em; /* 0.08px */
```

Comment:
maybe make a note that the letter-spacing px conversions are for font-size 16 px. it'll be different based on the current font-size since the unit is `em`